manifests/TimKosse/FileZillaClient/3.48.1.yaml

```
@@ -0,0 +1,15 @@
+Id: TimKosse.FileZilla
```

```suggestion
Id: TimKosse.FilezillaClient
```

Change the `Id` to `TimKosse.FilezillaClient` will fix this issue